cmd/generate/config/rules/kubernetes.go

```diff
@@ -15,6 +17,8 @@ func KubernetesSecret() *config.Rule {
 		Keywords: []string{
 			"kind: Secret",
 		},
+		// Kubernetes secrets are always yaml files, we limit to common yaml-endings to make this rule more safe!
+		Path: regexp.MustCompile(`\.(yaml|yml)$`),
```

Comment:
Can be simplified to:
```regex
\.ya?ml$
```

Reply:
I thinks this simplification reduces the readability for people that are not to familiar with regex. You still want me to compress it?

Reply:
It's the most straightforward and 'correct' way. Personally, I'd prioritize maintainability over readability to people not familiar with Regex.

Though it's up to zricethezav. I'm just some guy.

Reply:
Incidentally, it may need a [case-insensitive flag](https://stackoverflow.com/a/9655186) at the front.

```regex
(?i)\.ya?ml$
```